apps/sms/create_bridge_lib.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+###############################################################################
+
+# Fetch threads library and rename by using bower and browserify
+
+
+###############################################################################
+# variables
+bower_path="bower_components"
+threads_repo="gaia-components/threads#master"
+threads_index="$bower_path/threads/index.js"
+module_name="bridge"
+module_path="lib/$module_name.js"
+
+
+###############################################################################
+# Main script
+
+bower install $threads_repo
+
+browserify $threads_index --standalone $module_name > $module_path
+
+rm -rf $bower_path
```

Comment:
I think we could put it in a `apps/sms/.gitignore` file instead of deleting. What do you think?